src/main/resources/application.yml

```diff
@@ -22,12 +22,16 @@ zeebe:
 timer: "PT30S"
 
 channel:
-  host: "http://localhost:5001"
+  host: "http://localhost:8081"
```

Comment:
is this port number change just for testing? are we expecting channel to run on 8081?

Reply:
Depends on which spring port we are running channel.